src/gestures/drag/VisualElementDragControls.ts

```diff
@@ -102,6 +109,16 @@ export class VisualElementDragControls {
         y: 0.5,
     }
 
+    externalAxes: {
+        x?: MotionValue<number>
+        y?: MotionValue<number>
+    } = {}
+
+    private originPoint: {
+        x?: number
+        y?: number
+    } = {}
```

Comment:
Is there a better way to record the current scroll when using external values to inform subsequent drags?